decidim-demographics/app/controllers/decidim/demographics/admin/publish_responses_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Demographics
+    module Admin
+      class PublishResponsesController < Admin::ApplicationController
```

Comment:
```suggestion
      # Mind that even though the name of the controller, currently we don't allow the publication.
      # This is only for showing to the admin the visualization of the responses
      class PublishResponsesController < Admin::ApplicationController
```

Reply:
I think we should left this documented somewhere. I'm hesitant if we should just rename the controller/views/route to better reflect the current intent of this feature (and once/if we allow publication then we rename it again)